otter/test/convergence/test_service.py

```diff
@@ -1366,6 +1372,43 @@ def plan(*args, **kwargs):
             perform_sequence(self.get_seq() + sequence, self._invoke(plan)),
             ConvergenceIterationStatus.Stop())
 
+    def test_launch_stack_config(self):
+        lc = {'args': {'stack': {'stack_name': 'foo'}},
+              'type': 'launch_stack'}
+
+        self.manifest = {
+            'state': self.state,
+            'launchConfiguration': lc,
+        }
+
+        self.gsgi_result = (self.group, self.manifest)
+        self.gacd_runner = lambda i: {'stacks': []}
+
+        def plan(*args, **kwargs):
+            return []
+
+        executor = attr.assoc(launch_stack_executor,
+                              gather=intent_func('gacd'),
+                              plan=plan)
+
+        seq = [
+            parallel_sequence([]),
```

Comment:
I added a comment for it. Yes, it was because there were no steps, but it was for logging the steps.